src/actboulder.cpp

```diff
@@ -507,7 +568,7 @@ void actBoulder(Entity* my)
 						{
 							continue;
 						}
-						if ( boulderCheckAgainstEntity(my, entity) )
+						if ( BoulderCheckAgainstEntity(my, entity) )
```

Comment:
Compile error in VS, needs 3 arguments:
Error	C2660	'BoulderCheckAgainstEntity': function does not take 2 arguments	

bool BoulderCheckAgainstEntity(Entity* const my, Entity* const entity, const bool bWasBoulderPushed)